core/cmd/lambda/main.go

```diff
@@ -95,11 +104,40 @@ func handler(
 			), err
 		}
 
+		userID := readUserID(req.Headers)
+		requestID := readRequestID(ctx)
+		if err := clientStorage.WritePrompt(
+			ctx, core.UserInput{
```

Comment:
**suggestion** for better readability, I would like to suggest creating a var to hold this core.UserInput value there.

Reply:
Thanks for your suggestion! I gladly apply it.